x-pack/plugins/transform/common/utils/alerts.ts

```diff
@@ -13,7 +13,10 @@ export function getResultTestConfig(config: TransformHealthRuleTestsConfig) {
       enabled: config?.notStarted?.enabled ?? true,
     },
     errorMessages: {
-      enabled: config?.errorMessages?.enabled ?? true,
+      enabled: config?.errorMessages?.enabled ?? false,
+    },
+    healthCheck: {
+      enabled: config?.healthCheck?.enabled ?? true,
```

Comment:
For an earlier rule I'd created, where I'd disabled the check for error messages, the health check option will be enabled. As this check is replacing the error message check, do you think we should set this to `false` if `errorMessages` is not enabled? As things stand, this option is getting silently enabled where before I had the error message check turned off.

<img width="359" alt="image" src="https://user-images.githubusercontent.com/7405507/223403005-c05a5675-150c-4449-bc5e-35f71984c6bf.png">

Reply:
Updated in https://github.com/darnautov/kibana/commit/798f2367373bf8f36a2c8399735279a1564bc939